src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloudWindowsIT.java

```diff
@@ -268,6 +282,39 @@ public void testSnapshotCreated() throws Exception {
     }
   }
 
+  @Test(timeout = 500000)
```

Comment:
This class is pretty cluttered as is, wouldn't it make more since for this test to live in ComputeEngineCloudIgnoreProxyIT?

Reply:
I've been considering this. The first reason I put this here is because he windows tests take considerably longer than the rest of the integration tests, so I have been running them separately. There is also still a considerable amount of windows-specific setup required, and I could re-use that setup by adding my test here. I plan to address parallelizing the windows ITs in a separate PR.

Reply:
Makes sense. Could you leave a TODO so that we don't forget it.